src/xrc/src/exchanges.rs

```diff
@@ -483,6 +483,45 @@ impl IsExchange for CryptoCom {
     }
 }
 
+/// Bitget
+#[derive(Deserialize)]
+struct BitgetResponse {
+    data: Vec<(String, String, String, String, String, String, String)>,
+}
+
+impl IsExchange for Bitget {
+    fn get_base_url(&self) -> &str {
+        "https://api.bitget.com/api/v2/mix/market/history-candles?symbol=BASE_ASSETQUOTE_ASSET&granularity=1m&productType=usdt-futures&startTime=START_TIME&endTime=END_TIME"
```

Comment:
It looks like this is using the Bitget's futures API. I'm not sure if there is much difference between the two, but I think we want to use the spot API.

Reply:
Updated to the spot API